### PR TITLE
Text edit + Postdoc addition

### PR DESCRIPTION
**Pull request by droush**
I did a run through of the text and changed some things for clarity. I also added the postdoc section under People and added my information.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -17,9 +17,9 @@ header:
...
+We are the Qiyun Lab at the Biodesign Center for Fundamental and Applied Microbiomics ([CFAM](https://biodesign.asu.edu/cfam)) at Arizona State University ([ASU](https://www.asu.edu/)). We study the applications of data science for microbiomics and multiomics analyses. We are interested in the development of computational tools and resources to improve microbiome data analytics, the study of various host and environmental microbiomes and their influence on human health, and the investigation of the evolution of microbes and their hosts, with a focus on vertical and horizontal genetic connections between organisms. We strive to expand understanding of the microbial world through its multiple levels: genes, genomes, lineages, communities, hosts, and environments.
```
Comment (qiyunzhu) [inline index.md:22]: Why add "applications of"? I thought that "application" refers to using existing tools to solve biological questions, whereas the "development" / "theory" / "methodology" sides are also important. That's why the name of our center is called "CFAM".

File: people.md [hunk 2]
```diff
@@ -4,14 +4,27 @@ pi:
...
         <br /><br />
         [<i class='fa fa-envelope-square fa-2x'></i>](mailto:Qiyun.Zhu@asu.edu)&nbsp;&nbsp;
         [<i class='ai ai-google-scholar-square ai-2x'></i>](https://scholar.google.com/citations?user=mjkV0pUAAAAJ)&nbsp;&nbsp;
         [<i class='ai ai-orcid-square ai-2x'></i>](http://orcid.org/0000-0003-0490-1175)&nbsp;&nbsp;
         [<i class='fab fa-github-square fa-2x'></i>](https://github.com/qiyunzhu/)&nbsp;&nbsp;
         [<i class='fab fa-linkedin fa-2x'></i>](https://www.linkedin.com/in/qiyunzhu/)&nbsp;&nbsp;
         [<i class='ai ai-cv-square ai-2x'></i>](assets/files/QiyunZhu_CV_Aug2020.pdf)
+
+postdoc:
```
Comment (qiyunzhu) [inline people.md:15]: Good job! I am impressed by that you figured out the correct way of coding Jekyll front matter only a few moments since you started to play with GitHub Pages.

File: people.md [hunk 3]
```diff
@@ -4,14 +4,27 @@ pi:
...
+  - title: Daniel Roush
+    image_path: assets/images/people/roush-outdoors.png
+    excerpt: >
+        I am a Postdoctoral Scholar in the Center for Fundamental and Applied Microbiomics at the Biodesign Institute located at Arizona State University. Before joining the Qiyun Lab, I completed my dissertation work under the tutelage of [Dr. Ferran Garcia-Pichel](http://garcia-pichel.lab.asu.edu/labo/) where I examined endolithic phototrophs and their accompanying microbiomes. My (micro)biological training has spanned a wide range of topics including synthetic biology, microbial fuel cells, industrial applications of anaerobic extremophiles, and extreme environmental microbiomes. Most recently, I have led the development of databases and tools to help with rapid analysis of environmental microbiomes ([Cydrasil]( https://github.com/FGPLab/cydrasil)). My current research goals are focused on developing and adapting new technologies for use in microbiome analysis and making those new tools easily accessible for every scientist.
```
Comment (qiyunzhu) [inline people.md:19]: Very impressive self-introduction! Undoubtedly you are a competitive microbiologist @droush ! I have a few comments but feel free to ignore since I could likely be wrong about English:

1. Consider moving "and extreme environmental microbiomes" up to make logic more sound?
2. accessible "for" => "to" every scientist
3. `[Cydrasil]( https://`: Watch for that extra whitespace! It could break the link.
4. "research goals are focused...": "goals" and "focused" are duplicate?

File: research.md [hunk 4]
```diff
@@ -19,41 +19,41 @@ amazon:
...
+The rapid growth of microbiome data yields great opportunities to achieve higher accuracy, greater predictive power and novel insights into biology; at the same time, it poses great challenges in computation and statistics imposed by many dimensions of data and metadata. Advancing microbiome science demands fundamental innovations in both methods and applications, and can only be achieved by integrating wisdom from multiple disciplines. At the [Qiyun Lab](https://qiyunlab.github.io/), we are exploring a wide range of questions that span both the computational (community ecology to metagenome assembly) and the biological (the origin of life to the transmission of pathogenicity). We strive to push the frontier of microbiomics by combining the development of bioinformatics tools and the investigation of meta-omics datasets. We are dedicated to collaborative, reproducible and open-source science.
```
Comment (qiyunzhu) [inline research.md]: Maybe add "e.g." to the two parentheses, and replace "to" with "and"?

File: research.md [hunk 5]
```diff
@@ -19,41 +19,41 @@ amazon:
...
+Microbiome research has long benefitted from [phylogeny](https://en.wikipedia.org/wiki/Phylogenetic_tree) which makes sense of high-dimensional data by using a hierarchical representation of their relationships. Algorithms like UniFrac and Faith's PD along with databases like Greengenes and Silva have been widely adopted to describe phylogenetic relationships in modern high-throughput 16S rRNA gene amplicon studies. However, in the rapid-growing field of shotgun metagenomics, the utilization of phylogeny is still rare and primitive. Most studies are limited to the analysis of lower-dimensional genus- or species-level taxonomic units by using non-hierarchical methods, or rely on the taxonomy tree, which is inherently less accurate and more error-prone than explicit phylogeny.
```
Comment (qiyunzhu) [inline research.md]: Good try in elaborating the use of phylogeny for 16S analysis. But I am not sure if "phylogenetic relationships" is accurate. The purpose of those metrics and dbs are not to describe the evolutionary relationships of microbes, but to use the evolutionary relationships among microbes to describe the structure of whole communities. Also, I don't necessarily think we need to emphasize the "modern", "high-throughput" side of 16S, which is so 5-year-old...

File: research.md [hunk 6]
```diff
@@ -19,41 +19,41 @@ amazon:
...
 
 
 ## Diversity, Vertical and Horizontal Evolution of Microbes
 
-The natural history of living organisms is among the most fundamental and long-standing questions in science. Culture-independent methods especially metagenomics have enabled many exciting discoveries of novel microbial groups in recent years, leading to repeated re-writings of the tree of life, and yielding markers and products of medical, environmental and technological importance.
+The natural history of living organisms is among the most fundamental and long-standing questions in science. Culture-independent methods, especially metagenomics, have enabled many exciting discoveries of novel microbial groups in recent years. This has  led to repeated re-writings of the tree of life, and yielding markers and products of medical, environmental and technological importance.
```
Comment (qiyunzhu) [inline research.md]: Redundant whitespace.

File: research.md [hunk 7]
```diff
@@ -19,41 +19,41 @@ amazon:
...
+The evolutionary "tree" of microbes is often confounded by the discrepancies among individual genes trees due to the prevalence of horizontal gene transfer events ([HGTs](https://en.wikipedia.org/wiki/Horizontal_gene_transfer)). HGTs play vital roles in the adaptation of pathogens to host environments and the dispersal of virulence and antibiotic resistance. We develop computational methods for HGT identification, including [HGTector](https://github.com/DittmarLab/HGTector), an algorithm that has been shown effective in multiple studies, as well as more comprehensive methods incorporating similarity, composition, context, phylogeny and other characteristics indicative of HGTs. Our goal is identify the true “web of life”: a network structure incorporating both vertical and horizontal evolutions.
```
Comment (qiyunzhu) [inline research.md]: ...goal is **to** identify...

File: research.md [hunk 8]
```diff
@@ -19,41 +19,41 @@ amazon:
...
 
 
 ## Microbiome Interacting with Hosts and Environments
 
-Microbial communities interact with their environments -- either host-associated (human gut, oral, skin, etc.) or free-living (soil, concrete, deep sea, etc.). Such interactions are hiding under the complexity of biological big data. It is increasingly needed yet challenging for scientists to capture correlations among one to multiple omics datasets with hundreds to thousands of dimensions, and dozens to hundreds of metadata fields coded in diverse formats.
+Microbial communities interact with their environments -- either host-associated (human gut, oral, skin, etc.) or free-living (soil, carbonate, deep sea, etc.). Such interactions are hiding under the complexity of biological big data. Analysis of biological big data is increasingly needed yet extremely challenging for scientists. Capturing correlations among multiple omics datasets with hundreds to thousands of dimensions, and dozens to hundreds of metadata fields coded in diverse formats is a monumental task that only recently scientists have begun to understand how to approach.
```
Comment (qiyunzhu) [inline research.md:51]: Looks better!

**Review by qiyunzhu — COMMENTED** [review]
@droush Great job! I added a few comments. Also cc'ing @yujulian0168

**Comment by qiyunzhu** [comment]
PS: I was trying to modify the pages while (without noticing that) you were submitting a pull request. This causes some merge conflicts which you may need to resolve. Conflicts are common. One thing you can do to avoid it is that you submit pull requests to the qiyunlab repo, not my repo. @droush

**Comment by droush** [comment]
> PS: I was trying to modify the pages while (without noticing that) you were submitting a pull request. This causes some merge conflicts which you may need to resolve. Conflicts are common. One thing you can do to avoid it is that you submit pull requests to the qiyunlab repo, not my repo. @droush

My mistake. I didn't see the other repo. I'll fork it and edit from there moving forward.

File: index.md (hunk 1 above)
Comment (droush) [reply index.md:22]: How it reads to be right now is saying that we study data science derived from microbiomics and multiomics. I'm not sure if that was the original intent of the sentence.

My change flips the meaning. In this instance, "applications of" refers using any form of data science to solve problems. The meaning is that we take the principles of data science and use them to solve microbiomics and multiomics problems. So applications in this context is purely describing tangible use cases of data science principles. You could also rephrase it to something like "We develop and study applications of...". 

As for CFAM, you are correct from the meaning of the title. Fundamental = basic science, Applied can mean using existing tools to solve problems. But applied also has a broader meaning where you are taking that basic science and applying it to solve real world problems. For example, we make a finding about the gut microbiome after an omics analysis and you "apply" those findings to a medical treatment. In this instance apply/application means we are taking something we learned and using that to solve a problem.

File: people.md (hunk 2 above)
Comment (droush) [reply people.md:15]: Thanks! To be fair, you did a great job coding the whole thing and I could just work from your framework. Though it did take me a good hour to get Jekyll working on WSL2...

File: people.md (hunk 3 above)
Comment (droush) [reply people.md:19]: 1. I wrote it chronologically, but I could see this working instead in the sentence about the FGP lab. 
2. Interchangeable in this case, but "for" implies "for the good of"
3. Thanks for catching that. I remember seeing it and thought I fixed it.
4. In this case focus is the verb describing what my research goals are doing. I could rephrase to something like "My current research goals include developing..."

File: research.md (hunk 4 above)
Comment (droush) [reply research.md]: E.g. is good. I left the to to keep your phrasing/cadence with the sentence structure but that swap works as well.

File: research.md (hunk 5 above)
Comment (droush) [reply research.md]: Yeah now that I reread it, it is not correct. I'll rewrite for clarity. Modern high-throughput is habit at this point.

File: research.md (hunk 6 above)
Comment (droush) [reply research.md:38]: > @droush Great job! I added a few comments. Also cc'ing @yujulian0168

@qiyunzhu Responded to comments and making changes. Would you prefere if I fork the qiyunlab repo and do a pull request there instead?

**Comment by droush** [comment]
> @droush Great job! I added a few comments. Also cc'ing @yujulian0168

@qiyunzhu Responded to comments and making changes. Would you prefere if I fork the qiyunlab repo and do a pull request there instead?

File: people.md (hunk 2 above)
Comment (qiyunzhu) [reply people.md:15]: We can meet and go over the Jekyll thing, including how to run a Jekyll server in your own laptop so that you can preview your edits!

File: people.md (hunk 3 above)
Comment (qiyunzhu) [reply people.md:19]: Thanks for the explanation!

File: index.md (hunk 1 above)
Comment (qiyunzhu) [reply index.md:22]: Thanks for detailing your thoughts on the wording. I see your point about "application". I am still not fully sold though. Because we _not only_ use data science principles to solve problems, but we also _create_ data science principles themselves, the latter of which (I hope!) differentiate us from people who use existing tools. More specifically, we develop equations, algorithms, programs and protocols which other people can take to make findings from their microbiome and meta-omics datasets. Therefore I just used a single "data science" phrase.

(Maybe you made edits but I couldn't see from the pull request. Sorry... See my comment in the end.)

**Comment by qiyunzhu** [comment]
Hi @droush Not a problem this time! Just resolve conflicts locally and proceed with the current PR. Anyway it is just a website and we don't have to edit on a daily base. :)

File: people.md (hunk 2 above)
Comment (qiyunzhu) [reply people.md:15]: Sorry, it seems that you already got Jekyll running in your computer. Good job!

**Comment by qiyunzhu** [comment]
Hi @droush Had another chance to work on this PR? If you are not convenient recently, I can proceed to resolve conflicts and merge, then edit from my end. What do you think?

**Comment by droush** [comment]
> Hi @droush Had another chance to work on this PR? If you are not convenient recently, I can proceed to resolve conflicts and merge, then edit from my end. What do you think?

I was planning on taking care of it today. I'll have the conflicts resolved and edits incorporated some time this afternoon.

**Comment by droush** [comment]
@qiyunzhu Should be good to go here I think.

**Comment by qiyunzhu** [comment]
@droush Thanks for addressing most of my comments! I think it is good to go. I will merge and do some minor tweak and ping @yujulian0168 so that she can add her information following the example.
